src/bridges.ts

```diff
@@ -132,6 +133,16 @@ export const supportedBridges: Array<Bridge> = [
     // explorerUrl: '',
     // analyticsUrl: '',
   },
+  {
+    key: BridgeTool.across,
+    name: 'AcrossV2',
+    logoURI:
+      'https://raw.githubusercontent.com/lifinance/types/main/src/assets/icons/bridges/acrossv2.png',
```

Comment:
This file needs to be added I guess? :)

Reply:
Oh, I thought I had committed it. Nice catch!